lib/gateway/Shard.js

```diff
@@ -456,17 +457,21 @@ class Shard extends EventEmitter {
                     /**
                     * Fired when a message is created
                     * @event Client#messageCreate
-                    * @prop {Message} message The message
+                    * @prop {Message} message The message. Can be partial with only the channel_id if the channel is not cached
```

Comment:
The message itself won't be partial, the channel will, and that should be documented in the Message class